std_msgs/Header header

int32 network_id
int32 msg_id
int32[] data_bytes
